src/workloads/execution/DensifyTimeseriesCollection.yml

```diff
@@ -14,7 +14,7 @@ GlobalDefaults:
     options: {name: "numeric"}
   smallStepSize: &smallStep {^RandomInt: {min: 3, max: 3}}
   largeStepSize: &largeStep {^RandomInt: {min: 8, max: 8}}
-  explicitBounds: &bounds [{^RandomInt: {min: 5000, max: 15000}}, {^RandomInt: {min: 20000, max: 40000}}]
+  explicitBounds: &bounds [{^RandomInt: {min: 1000, max: 50000}}, {^RandomInt: {min: 10000, max: 20000}}]
```

Comment:
Not sure this will work either. The lower bound does have to be below the upper bound. I was getting at just having the upper bound start below the value in the documents.

Reply:
Oh ha, that's a typo. Supposed to be 5k I'll change